test/integration/document/Thumbnails.e2e.test.js

```diff
@@ -1,16 +1,28 @@
 // <reference types="Cypress" />
 describe('Preview Document Thumbnails', () => {
     const token = Cypress.env('ACCESS_TOKEN');
-    const fileId = Cypress.env('FILE_ID_DOC');
+    const fileId = Cypress.env('FILE_ID_DOC_LARGE');
     const THUMBNAIL_SELECTED_CLASS = 'bp-thumbnail-is-selected';
 
     /* eslint-disable */
-    const getThumbnail = (pageNum) => cy.get(`.bp-thumbnail[data-bp-page-num=${pageNum}]`);
+    const getThumbnail = (pageNum) => {
+        return cy
+            .get(`.bp-thumbnail[data-bp-page-num=${pageNum}]`, { timeout: 5000 })
```

Comment:
What's the default timeout? May be worth raising it to this value if it's below. We have several timeouts declared already in other tests.

Reply:
Default command timeout is 4s, I'll try overriding this

Reply:
It looks like the `defaultCommandTimeout` in Elements is 8 seconds, so you could definitely bump this one as well.